app/src/main/java/com/keylesspalace/tusky/ViewMediaActivity.kt

```diff
@@ -125,7 +125,7 @@ class ViewMediaActivity : BaseActivity(), ViewImageFragment.PhotoActionsListener
             AvatarImagePagerAdapter(supportFragmentManager, avatarUrl)
         }
 
-        viewPager.adapter = adapter
+        viewPager.adapter = adapter as PagerAdapter
```

Comment:
no cast needed

Reply:
Interestingly enough, the first time I wrote it it was fine (it resolves to existential - union of two types which is not representable in Kotlin because it's not representable in Java I guess) but when I was editing code before sending I got errors. Now I don't get them again. Weird, maybe something didn't conform to something.